node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -89,6 +98,51 @@ func NewOsDeviceConnectivityHelperScsiGeneric(executer executer.ExecuterInterfac
 	}
 }
 
+func (r OsDeviceConnectivityHelperScsiGeneric) GetMpathdOutputByVolumeIds(volumeUids []string) (string, error) {
+	return r.Helper.GetMpathdOutputByVolumeId(volumeUids)
+}
+
+func (r OsDeviceConnectivityHelperScsiGeneric) GetMpathDeviceName(volumePath string) (string, error) {
+	procMountsFileContent, err := ioutil.ReadFile(procMountsFilePath)
+	if err != nil {
+		return "", err
+	}
+	procMounts := string(procMountsFileContent)
+
+	return r.Helper.GetMpathDeviceNameFromProcMounts(procMounts, volumePath)
+}
+
+func (r OsDeviceConnectivityHelperScsiGeneric) GetMatchingVolumeIdToMpathName(mpathdOutput string,
+	mpathDeviceName string, volumeUids []string) (string, error) {
+
+	volumeId, _ := r.Helper.GetVolumeIdByMpathdName(mpathDeviceName, mpathdOutput)
+
+	dmPath := filepath.Join(devMapperPath, filepath.Base(strings.TrimSpace(mpathDeviceName)))
+	if volumeId != "" {
+		if r.Helper.IsdmPathMatchVolumeIdsWithoutErrors(dmPath, volumeUids) {
```

Comment:
```suggestion
		if r.Helper.IsDmPathMatchesVolumeIds(dmPath, volumeUids) {
```
I think `WithoutErrors` is implied by the fact that it doesn't return an error